### ISSUE TITLE
CMS for blog?

### ISSUE THREAD

**Issue by baslie** [vapid/vapid#32]
Hi! While reading docs I didn't understand is Vapid is suitable as a blog system with multiple pages and posts?

**Comment by srobbin** [comment]
@baslie Yes, take a look at the "Link to Repeating Content" section under "Sections". That describes how to link to an individual page, like something you'd use for a blog post.

https://docs.vapid.com/learn-more#sections

**Comment by siriokun** [comment]
Perhaps you can see example page that I created to create Blog posts,
https://github.com/siriokun/vp

In the dashboard, you just need to put working url to your post/page 😉

More examples soon!

**Comment by jcbbuller** [comment]
This is great! Maybe put this on a separate section in the documentation? I had the same problem trying to see if this would work for a repeated portfolio/blog site and I don't think we're alone.
